src/main/kotlin/me/clip/voteparty/cmds/CommandVoteParty.kt

```diff
@@ -29,16 +31,29 @@ internal class CommandVoteParty(override val plugin: VotePartyPlugin) : BaseComm
 {
 	
 	@Subcommand("addvote")
-	@Syntax("<amount>")
+	@Syntax("<amount> [player]")
 	@Description("Add Vote")
 	@CommandPermission(ADMIN_PERM)
-	fun addVote(issuer: CommandIssuer, @Default("1") amount: Int)
+	fun addVote(issuer: CommandIssuer, @Default("1") amount: Int, @Optional name: String?)
 	{
+		
 		if (amount <= 0)
 		{
 			return sendMessage(issuer, Messages.ERROR__INVALID_NUMBER)
 		}
 		
+		if (!name.isNullOrEmpty())
+		{
+			val user = party.usersHandler[name] ?: return sendMessage(issuer, Messages.ERROR__USER_NOT_FOUND)
+			
+			repeat(amount) {
+				val event = VoteReceivedEvent(user.player())
+				server.pluginManager.callEvent(event)
```

Comment:
Please inline this, theres no need for it to be separated

Reply:
Done!